dist/docker-images/ziti-controller/compose.test.yml

```diff
@@ -0,0 +1,91 @@
+volumes:
+  ziti-login:
```

Comment:
why is this volume even needed? it just adds complexity to this compose file and I don't see the value